cobra/cmd/project.go

```diff
@@ -92,7 +92,9 @@ func NewProjectFromPath(absPath string) *Project {
 	}
 
 	p := new(Project)
-	p.absPath = strings.TrimSuffix(absPath, findCmdDir(absPath))
+
+	cmdDir := findCmdDir(absPath)
```

Comment:
Since `cmdDir` is used only once, these two lines can be merged:

`p.absPath = strings.TrimSuffix(absPath, findCmdSuffix(findCmdDir(absPath)))`